apps/arena/lib/arena/game_updater.ex

```diff
@@ -148,6 +148,32 @@ defmodule Arena.GameUpdater do
     {:noreply, state}
   end
 
+  def handle_info({:recharge_stamina, player_id}, state) do
+    player = Map.get(state.game_state.players, player_id)
+
+    player =
+      case player.aditional_info.available_stamina do
+        3 ->
```

Comment:
I guess ideally this should be the `base_stamina` from the character

Reply:
Yes!